test/Scrabble/Move/MoveTests.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Scrabble.Move.MoveTests where
+
+import Test.Framework.Providers.HUnit
+import Test.Framework.TH
+import Test.HUnit
+import TestHelpers
+import Scrabble.Move.Move
+import Scrabble.Board.Board
+import Scrabble.Bag
+
+
+case_create_move =
+    let rack       = Rack $ map fromLetter [O, P, T]
+        (Right wp) = makeWordPut "TOP" Vertical (7,7) []
+        expected = do
+            (board',score) <- wordPut standardValidation newBoard wp dict
+            return $ Move wp score (Rack []) board'
+
+    in createMove newBoard rack wp dict @?= expected
+
+
+case_create_move_single_blank =
+    let rack       = Rack $ map fromLetter [O, P, Blank]
+        (Right wp) = makeWordPut "_OP" Vertical (7,7) ['T']
+        expected = do
+            (board',score) <- wordPut standardValidation newBoard wp dict
+            return $ Move wp score (Rack []) board'
+
+    in createMove newBoard rack wp dict @?= expected
```

Comment:
All of these tests have basically the same shape. I think you could pull out a lot of this code into one common function, and reduce the amount of code in each of the tests. I strive to make tests as small and as readable as possible, even if they delegate to big ugly functions.